dao/pom.xml

```diff
@@ -8,13 +8,18 @@
         <version>1.0.0</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-
     <artifactId>dao</artifactId>
+    <packaging>jar</packaging>
```

Comment:
please maven for managing dependencies, do not import jars directly

Reply:
@valevaty98 done